src/routes/_auth.services_.$serviceId.tsx

```diff
@@ -36,26 +38,69 @@ function ServiceView() {
     authUserIsAdmin,
     service,
     serviceTypes: services,
-    refreshServices
+    refreshServices,
   } = Route.useLoaderData();
 
   const queryClient = useQueryClient();
-  queryClient.invalidateQueries();
-
   const [showEditServiceModal, setShowEditServiceModal] = useState(false);
+  const [availableSlots, setAvailableSlots] = useState<number[]>([]);
 
   const { data: guestsSlotted, isPending: isSlotsPending } = useQuery({
     queryFn: () => fetchServiceGuestsSlotted(service.service_id),
-    queryKey: ["guestsSlotted"],
+    queryKey: ["guestsSlotted", service.service_id],
```

Comment:
Adding the service ID ensures that the data will refresh if we switch to a new service page.